### ISSUE TITLE
Versionbot: VB does bizarre things if the PR has a prerelease version bump

### ISSUE THREAD

**Issue by pimterry** [resin-io-modules/resin-procbots#348]
In https://github.com/resin-io-modules/resin-device-init/pull/25, the PR contained a version bump already (from `4.0.1` to `4.1.0-rc1`), because I've been testing it integrated downstream before merging, and I needed a prerelease version with which to do so.

I probably should've remembered to remove this before merging, but it'd be nice if I didn't have to, or if VB complained up front. Instead, VB created a commit for the previous version number (`v4.0.1`), which did change the files to the right version (`4.1.0`), but put the changelog entry _underneath_ the latest version number, bizarrely. VB's commit is here: https://github.com/resin-io-modules/resin-device-init/pull/25/commits/65b17f797aad6685364e7b2c8c5cccfc59557368. It then merged the PR, but failed to tag it, because it tried to tag with the old version that already exists.

This is a messy case and it'd be fine if VB rejected it, but it's a bit awkward that VB has instead merrily merged it, done some odd things en route, and left master in a broken state.

**Comment by lurch** [comment]
> put the changelog entry underneath the latest version number, bizarrely

That part is because your Changelog's header doesn't adhere to the [standard header](https://github.com/resin-io/process/blob/master/process/Commit_and_PR_Guidelines.md#versionbot-automatic-versioning-review-requirements-and-changelog-generation) which means the default value [here](https://github.com/resin-io/versionist/blob/master/versionist.conf.js#L28) skipped too many lines, and thought the `## v4.0.1 - 2017-11-21` was simply part of the header.

@jhermsmeier encountered the same thing on one of the Etcher-related module repos a while ago ;-)

**Comment by pimterry** [comment]
@lurch true. I thought that'd been fixed, but clearly not. Good spot, thanks!

I think the issue with the wrong version tag being used is separate, and still remains, but I'll get this sorted too.
